eng/helix/content/runtests.cmd

```diff
@@ -38,25 +33,25 @@ if not errorlevel 1 (
 
 set exit_code=0
 
-REM Run non-flaky tests first
-REM We need to specify all possible Flaky filters that apply to this environment, because the flaky attribute
-REM only puts the explicit filter traits the user provided in
-REM Filter syntax: https://github.com/Microsoft/vstest-docs/blob/master/docs/filter.md
-set NONFLAKY_FILTER="Flaky:All!=true&Flaky:Helix:All!=true&Flaky:Helix:Queue:All!=true&Flaky:Helix:Queue:%HELIX%!=true"
-echo Running non-flaky tests.
-%DOTNET_ROOT%\dotnet vstest %target% --logger:xunit --TestCaseFilter:%NONFLAKY_FILTER%
-if errorlevel 1 (
-    echo Failure in non-flaky test 1>&2
-    set exit_code=1
-    REM DO NOT EXIT
-)
-
-set FLAKY_FILTER="Flaky:All=true|Flaky:Helix:All=true|Flaky:Helix:Queue:All=true|Flaky:Helix:Queue:%HELIX%=true"
-echo Running known-flaky tests.
-%DOTNET_ROOT%\dotnet vstest %target% --TestCaseFilter:%FLAKY_FILTER%
-if errorlevel 1 (
-    echo Failure in flaky test 1>&2
-    REM DO NOT EXIT and DO NOT SET EXIT_CODE to 1
+set NONQUARANTINE_FILTER="Flaky:All!=true&Flaky:Helix:All!=true&Flaky:Helix:Queue:All!=true&Flaky:Helix:Queue:%HELIX%!=true"
+if (%quarantined%==true) (
+    echo Running all tests (including quarantined).
```

Comment:
Sorry for the late response on this point. I'm not sure we want to rerun all of the unquarantined tests in this pipeline. That sounds significantly more wasteful that submitting empty jobs to Helix (hopefully those exit almost immediately).

Reply:
Hmm, @anurse won't this mean you would need to do extra Kusto magic in order to see information specifically about quarantined tests?

Reply:
the name of the test will have quarantined or not in it now, so the attributes don't matter as much.

i don't think there's much difference whether our jobs exit immediately or run for an additional 10-15 seconds, and the flaky runs are sort of wasteful entirely anyways since we are just running them to stress test our tests, seems like it doesn't hurt to just run everything on this optional run.

Reply:
> Hmm, @anurse won't this mean you would need to do extra Kusto magic in order to see information specifically about quarantined tests?

More runs of all tests is goodness here while we work on deflaking, I think. They give us more data as to whether a test is flaky. If a "not quarantined" test fails in the "quarantined tests" run, well.. it should probably be reviewed and quarantined ;).